### ISSUE TITLE
Doesn't allow photoprism hosted on local network with reverse proxy

### ISSUE THREAD

**Issue by AkshaySiramdas** [thielepaul/photoprism-mobile#113]
Hello all,

I've been using the app for the past 3 weeks. It shows the uploaded photos when direct local ip and port is used as the 'Photoprism URL' but doesnt show anything when I use the reverse proxied address as the URL. I think the issue might be due to certificatesand I hope there would be an option to select whether to trust the certificate or not.

**Comment by nibbles-bytes** [comment]
Do you have a local DNS server? Are you able to reach your reverse proxied site locally from a web browser?

**Comment by AkshaySiramdas** [comment]
Yes, I have a local PiHole as the DNS server and can reach via the browsers on both laptop and mobile.

**Comment by mmguero** [comment]
I have the same experience
